opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -1895,8 +1898,6 @@ namespace {
                     header.guide_rate_damping,
                     use_free_gas
                 };
-
-                this->updateGuideRateModel(guide_rate_model, report_step);
```

Comment:
Did you intend to remove this statement or did that just happen by accident?

Reply:
Ooops! 😬